.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# fdc3-maintainers will be default reviewers for everything in the repo unless a later match takes precedence
+* @finos/fdc3-maintainers
+
+# Any changes to docs (specs) should also be reviewed by fdc3-editors
+/docs/ @finos/fdc3-editors @finos/fdc3-maintainers
```

Comment:
@robmoffat Is fdc3-editors a private team at the org level or is it public?

Reply:
The other issue could be that is that the group as a whole doesn't have write access (individuals within the group might have it as being maintainers/admins).